### PR TITLE
[tests-only][full-ci]send nightly notification to builds channel

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
## Description
With this PR, the normal build notifications are sent to the `infinitescale` channel and nightly build notifications are sent to the `builds` channel. Previously all the build notifications were sent on `infinitescale` channel, due to which there might be a chance of missing some build status.

## Related Issue

- https://github.com/owncloud/ocis/issues/8794

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by saw-jan — APPROVED** [review]
LGTM

File: .drone.star [hunk 1]
```diff
@@ -1959,8 +1959,12 @@ def makeGoGenerate(module):
 
 def notify(ctx):
     status = ["failure"]
+    channel = ""
     if ctx.build.event == "cron":
         status.append("success")
+        channel = "builds"
+    else:
+        channel = config["rocketchat"]["channel"]
```
Comment (grgprarup) [inline .drone.star:1967]:
```suggestion
    channel = config["rocketchat"]["channel"]
    if ctx.build.event == "cron":
        status.append("success")
        channel = "builds"
```

Comment (individual-it) [reply .drone.star:1967]: can we put `"builds"` also in a config variable?

File: .drone.star [hunk 2]
```diff
@@ -153,6 +153,7 @@ config = {
     },
     "rocketchat": {
         "channel": "infinitescale",
+        "channel_builds": "builds",
```
Comment (phil-davis) [inline .drone.star:156]:
```suggestion
        "channel_cron": "builds",
```

This is the channel to report to for drone "cron" jobs (i.e. when the build event is "cron").
So I suggest naming it that.

File: .drone.star [hunk 3]
```diff
@@ -1959,8 +1960,10 @@ def makeGoGenerate(module):
 
 def notify(ctx):
     status = ["failure"]
+    channel = config["rocketchat"]["channel"]
     if ctx.build.event == "cron":
         status.append("success")
+        channel = config["rocketchat"]["channel_builds"]
```
Comment (phil-davis) [inline .drone.star:1966]:
```suggestion
        channel = config["rocketchat"]["channel_cron"]
```

**Review by phil-davis — APPROVED** [review]

**Review by ScharfViktor — APPROVED** [review]

**Pull request merged**